test/demo/recreate.inttest.ts

```diff
@@ -0,0 +1,144 @@
+import 'jest';
+
+import {Application, Window} from 'hadouken-js-adapter';
+import {WindowEvent} from 'hadouken-js-adapter/out/types/src/api/events/base';
+
+import {NotificationOptions, Notification, NotificationClosedEvent} from '../../src/client';
+
+import {createApp} from './utils/spawnRemote';
+import {isCenterShowing} from './utils/notificationCenterUtils';
+import * as notifsRemote from './utils/notificationsRemoteExecution';
+import {delay} from './utils/delay';
+import {fin} from './utils/fin';
+import {getToastIdentity} from './utils/toastUtils';
+import {assertDOMMatches, CardType} from './utils/noteCardUtils';
+
+const testManagerIdentity = {uuid: 'test-app', name: 'test-app'};
+
+const firstOptions: NotificationOptions = {
+    id: 'duplicate-test-1',
+    body: 'First Notification Body',
+    title: 'First Notification Title'
+};
+
+const secondOptions: NotificationOptions = {
+    id: 'duplicate-test-1',
+    body: 'Second Notification Body',
+    title: 'Second Notification Title'
+};
+
+describe('When creating a notification with an ID that already exists but different options', () => {
+    describe.each([true, false])('Center showing: %p', showCenter => {
+        let testApp: Application;
+        let testWindow: Window;
+
+        let existingNote: Notification;
+
+        beforeAll(async () => {
```

Comment:
Same comment as for `createCenter`

Edit: No, wait, this makes it make sense! (Though would probably still do it as - assert center is closed, open center if we want to, be a good citizen and close the center if we opened it)

Reply:
As we discussed offline, I've added an afterAll which will close the centre for the cases where we open it.